src/content/pages/pyladies-events.mdx

```diff
@@ -6,17 +6,50 @@ subtitle: Lunch, Workshops & Networking
 # PyLadies Events
 We’re excited to announce a range of events for underrepresented groups in computing this year! 🎉 Whether you're new to PyLadies or a long-time supporter, we warmly welcome you to join us and be part of our supportive community.
 
-## #IAmRemarkable
-Empower yourself in this two-hour workshop designed to help you celebrate your achievements and improve your self-promotion skills.
+![](./images/pyladies-events.jpg)
+
+## Self-defence Workshop
+A two-hours workshop where you can share your challenging experiences and learn how to defend yourself against inappropriate behaviour. This is a space to foster mutual support, understanding and effective strategies for personal safety. This session will be facilitated by a therapist.
+
+WHEN?
+Thursday, 11th July 2024, 10:30 to 12:30.
+
+WHERE?
+Prague Congress Centre (PCC), Open Space 2 (Rooms 223 + 224)
 
 ## PyLadies Lunch
 Join us for a special lunch event aimed at fostering community and empowerment in tech. Enjoy meaningful conversations and networking opportunities.
 
-## Self-defence Workshop
-A two-hours workshop where you can share your challenging experiences and learn how to defend yourself against inappropriate behaviour. This is a space to foster mutual support, understanding and effective strategies for personal safety. This session will be facilitated by a therapist.
+WHEN?
+Thursday, 11th July 2024, 12:30 to 14:00.
+
+WHERE?
+Prague Congress Centre (PCC), VIP lunch area.
+
+## #IAmRemarkable
+Empower yourself in this workshop designed to help you celebrate your achievements and improve your self-promotion skills.
+
+WHEN?
+Thursday, 11th July 2024, 15:30 to 17:00.
+
+WHERE?
+Prague Congress Centre (PCC), Open Space 2 (Room 224)
+
 
 ## Meet & Greet with PyLadies
 Take advantage of this unique networking opportunity to meet and connect with experienced members of PyLadies. Gain valuable insights, advice, and inspiration for your Python and life journeys.
 
+WHEN?
+Thursday, 11th July 2024, 15:30 to 17:00.
+
+WHERE?
+Prague Congress Centre (PCC), Open Space 2 (Room 223)
+
+
 # How To Apply?
-As capacity is limited, please apply soon. To reserve your seat and confirm your participation in these events, please fill in 👉🏽 [this form](https://docs.google.com/forms/d/e/1FAIpQLSdMIIK7jH7uzrBYbsLmetXBdcydAcdhpFxl_9K4jdRzdRx5zA/viewform) 👈 ASAP. We’ll send you a code for registration shortly after.
+Capacity is limited. Please apply soon!
+To reserve your seat and confirm your participation in these events, please fill in the form below. We’ll send you a code for registration shortly after.
```

Comment:
The form says that people need a conference ticket to participate. So "we will send you a code for registration" is for the specific event registration code, that can only be applied on top of a valid conference ticket?

I think it might be useful to put the conference ticket requirement on the page as well as on the form.

Reply:
AFAIK, there will be no registration code to be sent.
Instead, I agree that we should mention the need to have a conference ticket to participate in any of the activities.

Reply:
Okay, I removed that sentence and added a note at the beginning that the events are exclusive to ticket holders. Thanks for the tip!